### PR TITLE
[OPPRO-265] Pass spark unit test - add null test

### PR DESCRIPTION
**Pull request by jinchengchenghh**

### CODE REVIEW

**Comment by zhouyuan** [comment]
@zzcclp PTAL

**Review by zzcclp — APPROVED** [review]
LGTM, we will check these unit tests for ClickHouse Backend.

File: tools/build_velox.sh [hunk 1]
```diff
@@ -9,8 +9,8 @@ COMPILE_VELOX=OFF
 VELOX_HOME=${3:-/root/velox}
 ENABLE_EP_CACHE=OFF
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/jinchengchenghh/velox.git
```
Comment (zzcclp) [inline tools/build_velox.sh:12]: does is need to modify back to `oap-project` repo?

Comment (jinchengchenghh) [reply tools/build_velox.sh:12]: Yes, I asked @rui-mo to review velox PR https://github.com/oap-project/velox/pull/50. After it merged, I will change to oap-project and merge this one

Comment (rui-mo) [reply tools/build_velox.sh:12]: Merged the PR for Velox.

**Pull request merged**